citi-onboarding/HighBall
# Catalog props

## What I did

- Linked the catalog section with the local API
   
## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout main`
   1. `git pull`
   1. `git checkout catalog-props`
   1. `git pull origin catalog-props`
   1. `yarn install`
   1. `code .`
   1. `cd client`
   1. `yarn start`

1. Open a second terminal:
   1. `cd server`
   1. `sudo docker-compose up`

1. In your browser go to http://localhost:3000 and http://localhost:3001

1. Open Insominia and try to put a new card

1. Check if the data is sent to the correct places

1. In VScode open `./client`

1. Check the code

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description
